network/p2p/libp2pNode.go

```diff
@@ -729,7 +736,6 @@ func DefaultPubsubOptions(maxPubSubMsgSize int) []PubsubOption {
 		pubSubOptionFunc(pubsub.WithStrictSignatureVerification(true)),
 		// set max message size limit for 1-k PubSub messaging
 		pubSubOptionFunc(pubsub.WithMaxMessageSize(maxPubSubMsgSize)),
-		// no discovery
```

Comment:
I kept the comment in intentionally to highlight that we are no using any discovery mechanism :)